twitter.js

```diff
@@ -5,6 +5,8 @@ const querystring = require("querystring");
 const Stream = require("./stream");
 
 const getUrl = subdomain => `https://${subdomain}.twitter.com/1.1`;
+const getOauth = subdomain => `https://${subdomain}.twitter.com/oauth`;
```

Comment:
What if we added a second param to `getUrl`?

```
const getUrl = (subdomain, endpoint = '1.1') => `https://${subdomain}.twitter.com/${endpoint}`?
```

`endpoint` would default to `1.1`